vale-incubacao.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
```

Comment:
Not that relevant, but should we keep all the page names in english? 🤔 

What do you guys think?

Reply:
totally agree

or is it a requirement?

Reply:
I've the same opinion...

Reply:
I'll call it "co-financing"